addons/html_builder/static/src/builder.js

```diff
@@ -124,6 +124,8 @@ export class Builder extends Component {
                     can_display_toolbar: (namespace) => !["image", "icon"].includes(namespace),
 
                     // disable the toolbar for images and icons
+                    non_html_text_container_selectors:
```

Comment:
`non_html_text` is quite a convoluted concept. Does it mean an editable field that is of any type but `html`? The rest of the review will depend on the meaning of this concept, ie what you mean to do here exactly, not the exact definition that I can read in the selector :-P What do these things have in common?

Reply:
I see. Best in that case would be to give the selector a name that is more descriptive of its function rather than of its "implementation". The goal being that reader understands what purpose it serves so it's easy to know when to use it, and if they want to see what that entails technically (how such nodes are branded), they can go read the selector. That also would make it more future proof as the concept will remain but the implementation might change (for instance, we invent a new attribute: `data-textonly`).
Finally, that would solve my other complain, which is to have a concept defined by what it's _not_ (**non** html) rather than what it is.

Do you think something like `restricted_text_container_selectors` would describe it well?

Reply:
Side note: there's a report in the [pad](https://pad.odoo.com/p/mysterious-egg-builder-editor-bug):
> [MOU, chrome]: As a "Restricted Editor", I still can edit content using the text editing bar in edit mode (content won't be saved ofc).

This PR could at least help solve it right? By adding a definition of `restricted_text_container_selectors` that fetches the user's information?